src/util/psbt/map/global.rs

```diff
@@ -62,6 +69,7 @@ impl Map for Global {
         } = pair;
 
         match raw_key.type_value {
+            1u8 => {}
```

Comment:
This entire `insert_pair` method isn't called by anything since we don't use the `impl_psbtmap_consensus_decoding` macro on `Global`, currently. Not sure what to do here ...